README.md

```diff
@@ -107,6 +107,7 @@ Anyone who has been active in the foundation or one of its member projects, as d
 - Sendil Kumar (@sendilkumarn)
 - Tierney Cyren (@bnb)
 - Waleed Ashraf (@waleedashraf)
+- Sara Chipps (@sarajo)
```

Comment:
```suggestion
- Sara Chipps (@sarajo)
- Sendil Kumar (@sendilkumarn)
- Tierney Cyren (@bnb)
- Waleed Ashraf (@waleedashraf)
```

It _seems_ the list is alphabetically ordered by first name.

Reply:
I believe this is correct.